.github/workflows/build-new.yml

```diff
@@ -7,76 +7,49 @@ on:
   pull_request:
     types: [ assigned, opened, synchronize, reopened ]
   release:
-    types: [published, edited]
+    types: [ published, edited ]
   workflow_dispatch:
 
 jobs:
-  linux-build:
-    name: Wrapper Linux Build
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        pyversion: ["cp36-cp36m", "cp37-cp37m", "cp38-cp38", "cp39-cp39"]
-    steps:
-    - name: Checkout
-      uses: actions/checkout@v2
-      with:
-        submodules: recursive
-    - name: Build the Linux wheels
-      run: |
-        # must be run in docker, cannot be run as freestanding script for toolchain issues
-        # run on old CentOS. but it's CentOS, so it's yum(RPM) not apt. very old glibc. glibc API is back-compatible but not forward. See https://github.com/pypa/manylinux
-        # see https://github.com/varunagrawal/docker-images/blob/master/gtsam-manylinux/Dockerfile for an example
-        # container, but it does not have some necesssary boost packages --> varunagrawal/gtsam-manylinux:latest
-        sudo docker run --rm -e PLAT=manylinux2014_x86_64 -e PYTHON_VERSION=${{ matrix.pyversion }} -v `pwd`:/io quay.io/pypa/manylinux2014_x86_64 /io/package/build-wheels-linux.sh ${{ matrix.pyversion }}
-        # cleanup for custom runner
-        sudo chown -R $(whoami):$(whoami) .
-    - name: Archive wheels
-      uses: actions/upload-artifact@v2
-      with:
-        # we strip the version number from the artifact name
-        name: pycolmap-${{ matrix.pyversion }}-manylinux2014_x86_64
-        path: wheelhouse/pycolmap-*-${{ matrix.pyversion }}-manylinux2014_x86_64.whl
-
   mac-build:
     name: Wrapper macOS Build
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macos-11, macos-10.15]
-        pyversion: ["python@3.7", "python@3.8", "python@3.9"]
+        os: [ macos-latest ]
+        pyversion: [ "python@3.8", "python@3.9", "python@3.10", "python@3.11" ]
```

Comment:
Please re-add 3.7 or give a valid reason for removal.

Reply:
Because anaconda remove from his channel and colab etc upgrade to 3.8